R/performance-package.R

```diff
@@ -0,0 +1,27 @@
+#' \code{performance-package}
```

Comment:
Does this create problems?

```suggestion
#' \code{performance}
```

Reply:
yes it creates a warning in R CMD check:
> Rd files with duplicated alias 'performance':
>     'model_performance.Rd' 'performance-package.Rd'